packages/plans-grid-next/src/index.tsx

```diff
@@ -40,10 +40,39 @@ const WrappedComparisonGrid = ( {
 		onUpgradeClick,
 	} );
 
-	const classNames = classnames( 'plans-grid-next', 'plans-grid-next__comparison-grid' );
+	const gridContainerRef = useRef< HTMLDivElement | null >( null );
+	const gridSize = useGridSize( {
+		containerRef: gridContainerRef,
+		containerBreakpoints: new Map( [
+			[ 'small', 0 ],
+			[ 'smedium', 686 ],
+			[ 'medium', 835 ], // enough to fit Enterpreneur plan. was 686
+			[ 'large', 1005 ], // enough to fit Enterpreneur plan. was 870
+			[ 'xlarge', 1238 ],
+		] ),
+	} );
+
+	// const gridSize = useGridSize( {
+	// 	containerRef: gridContainerRef,
+	// 	containerBreakpoints: new Map( [
+	// 		[ 'small', 0 ],
+	// 		[ 'smedium', 835 ], // enough to fit Enterpreneur plan. was 686
+	// 		[ 'medium', 1005 ], // enough to fit Enterpreneur plan. was 870
+	// 		[ 'large', 1238 ],
+	// 	] ),
+	// } );
```

Comment:
Can this be removed?

Reply:
Yes, of course. They are the original grid "reactions" to the screen resizing (showing 2, 3, 4, 4+ columns on each accordingly)